### ISSUE TITLE
Not compileable on VS2015 machine

### ISSUE THREAD

**Issue by raymens** [fsprojects/FSharpLint#120]
Installed: VS2015 on a W10 machine including Microsoft Build Tools 2013.

The project **FSharpLint.Console** and **FSharpLint.Application** won't open in VS2015. Opening it causes the text: *The operation could not be completed* to appear. This is the only project I know that causes this on my machine.

This is the error I get when running `build.cmd`:
```
Created AssemblyInfo file "src/FSharpLint.Rules/AssemblyInfo.fs".
Finished Target: AssemblyInfo
Starting Target: RunFunctionalTests
Running build failed.
Error:
System.Exception: NUnit: cannot run tests (the assembly list is empty).
   at Fake.NUnitSequential.NUnit(FSharpFunc`2 setParams, IEnumerable`1 assemblie
s) in C:\code\fake\src\app\FakeLib\UnitTest\NUnit\Sequential.fs:line 22
   at FSI_0001.Build.clo@100-8.Invoke(Unit _arg5) in C:\Users\Raymen\Documents\G
itHub\FSharpLint\build.fsx:line 101
   at Fake.TargetHelper.runSingleTarget(TargetTemplate`1 target) in C:\code\fake
\src\app\FakeLib\TargetHelper.fs:line 481

---------------------------------------------------------------------
Build Time Report
---------------------------------------------------------------------
Target            Duration
------            --------
Clean             00:00:00.0018497
RestorePackages   00:00:01.8694183
AssemblyInfo      00:00:00.0399789
Total:            00:00:02.0775903
Status:           Failure
---------------------------------------------------------------------
  1) System.Exception: NUnit: cannot run tests (the assembly list is empty).
   at Fake.NUnitSequential.NUnit(FSharpFunc`2 setParams, IEnumerable`1 assemblie
s) in C:\code\fake\src\app\FakeLib\UnitTest\NUnit\Sequential.fs:line 22
   at FSI_0001.Build.clo@100-8.Invoke(Unit _arg5) in C:\Users\Raymen\Documents\G
itHub\FSharpLint\build.fsx:line 101
   at Fake.TargetHelper.runSingleTarget(TargetTemplate`1 target) in C:\code\fake
\src\app\FakeLib\TargetHelper.fs:line 481
---------------------------------------------------------------------
```

**Comment by duckmatt** [comment]
Thanks, I'm able to repeat the bug when running `build.cmd`. However all the projects appear to be opening for me in VS2015 Professional on Windows 10, what version of VS2015 are you opening it in?

**Comment by raymens** [comment]
Great!

The issue of the not opening projects was fixed by installing F# Tools 3.1.2. It wasn't actually those 2 projects, after looking into it a bit more I found out it were all the F# projects, so it had to be F# tooling I guessed.

Thanks!
